### PR TITLE
add-more-dpg-test-coverage-for-rlc

### PR DESCRIPTION
**Pull request by qiaozha**
This PR is trying to add more serviceDriven scenario test coverage for rlc code generator. 

We still don't cover customization scenario in this PR as this is not a scope for rlc GA.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,9 +31,8 @@
         "debug": "node --inspect-brk ./dist/src/main.js",
         "generate-swaggers": "ts-node ./test/utils/test-swagger-gen.ts",
         "rlc-generate-swaggers": "ts-node ./test/utils/test-swagger-gen.ts -- rlc",
-        "validate-version-tolerance": "npm run start-test-server:v2 & npm run generate-version-tolerance && npm run generate-version-tolerance:tests && npm run test-version-tolerance && npm run stop-test-server",
+        "validate-version-tolerance": "npm run start-test-server:v2 & npm run generate-version-tolerance && npm run test-version-tolerance && npm run stop-test-server",
```
[the remark this answers is not in the document]
Comment (qiaozha) [reply package.json]: Thanks, I merged your PR.

**Review by joheredi — APPROVED** [review]
Looks good, thanks for taking on this. Would it be possible to include in this PR calls to the customization endpoints to mark as covered? Those scenarios do not apply to RLC but it would be good to mark them as covered to remove noise

**Comment by qiaozha** [comment]
> Looks good, thanks for taking on this. Would it be possible to include in this PR calls to the customization endpoints to mark as covered? Those scenarios do not apply to RLC but it would be good to mark them as covered to remove noise

okay, let me see how to cover that.

**Review by joheredi — APPROVED** [review]

**Comment by qiaozha** [comment]
I see the test coverage in the dashboard is 100% now :)

**Pull request merged**